README.md

```diff
@@ -1,4 +1,8 @@
-# PushmiPullyu
+<p align="center">
+  <a href="https://github.com/ualbertalib/pushmi_pullyu/">
+    <img src="docs/images/pushmi-pullyu.png" alt="PushmiPullyu logo">
+  </a>
+</p>
```

Comment:
You could just simplify this and use straight markdown:
```markdown
![PushmiPullyu logo](docs/images/pushmi-pullyu.png)
```
But it looks kinda weird:
![image](https://cloud.githubusercontent.com/assets/1930474/26263000/172d61e6-3c94-11e7-9bc5-d66e50280851.png)

So decided to use some html to center this and make it look a bit nicer